docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Methods
+---
+
+The SDK provides the following methods for a smart account.
+
+## BiconomySmartAccountV2
+### Create
+
+The `create()` method is used to create an instance of the Biconomy Smart Account V2. This method takes a configuration object and returns the smart account instance.
+
+The addresses of ERC-4337 smart accounts follow a deterministic pattern. This enables you to get the address off-chain before the account is actually deployed.
+
+**Usage**
+
+```jsx
+let provider = new ethers.providers.JsonRpcProvider("rpcUrl");
+let signer = new ethers.Wallet("private key", provider);
+
+
+const smartAccount = await BiconomySmartAccountV2.create({
+  signer: signer,
+  chainId: ChainId.POLYGON_MUMBAI, // Specify the desired chain (e.g., Polygon Mumbai)
+  bundlerUrl: "", // bundler URL (required)
+  biconomyPaymasterApiKey: "", // Instance of the paymaster
+});
+
+```
+
+**Parameters**
+
+*required params are explicitly mentioned*
+
+- config (`object`, required): A `BiconomySmartAccountV2Config` object containing configuration options for creating the smart account.
+  - signer(`Signer`, required): The signer instance that will need to be passed. 
+  - chainId (`ChainId` enum, required): The identifier for the blockchain network. (e.g., ChainId.POLYGON_MUMBAI).
+  - bundlerUrl (`string`, required) OR bundler (`IBundler`, required) : bundler url which will be internally used to create bundler instance or the bundler instance. Bundler instance can also be used if one wants to customise the bundler. Refer to bundler [integration](./../Bundler/integration.mdx) for more details on bundler.
+  - biconomyPaymasterApiKey(`string`) OR paymaster (`IPaymaster`): one can either pass paymaster API key or the paymaster instance.
+  - entryPointAddress (`string`): default entry point address will be used if not passed, otherwise the passed address will be used.
+  - defaultValidationModule (`BaseValidationModule`): The default validation module to start with ( ECDSA ).
+  - activeValidationModule (`BaseValidationModule`): The active validation module to start with ( either ECDSA or Multi-chain ).
+  - rpcUrl (`string`): RPC URL of the chain
+  - index (`number`): index to create multiple smart accounts for an EOA
+
+**Returns**
+
+- `smartAccount` (`object`): An instance of the Biconomy Smart Account V2.
+
+:::info
+Building on Chiliz Mainnet or the Spicy Testnet? Note that the entry point address on this is different as it was deployed by us on the Biconomy team. The address of the entry point is : [0x00000061FEfce24A79343c27127435286BB7A4E1](https://scan.chiliz.com/address/0x00000061FEfce24A79343c27127435286BB7A4E1/contracts#address-tabs)
```

Comment:
we sould mention this near where we are passing entryPointAddress as DEFAULT_ENTRY_POINT_ADDRESS